README.md

```diff
@@ -1,56 +1,30 @@
-## AWS CloudFormation 101 - Workshop
+<h1 align="center">
+AWS CloudFormation - Workshop
+<br>
+    <a href="https://cfn101.workshop.aws"><img alt="Website" src="https://img.shields.io/website?down_color=red&down_message=down&up_color=green&up_message=up&url=https%3A%2F%2Fcfn101.workshop.aws"></a>
+    <a href="https://github.com/aws-samples/cfn101-workshop/actions"><img alt="GitHub Workflow Status" src="https://github.com/aws-samples/cfn101-workshop/workflows/Unit%20Tests/badge.svg"></a>
+</h1>
 
-[![Website cfn101.workshop.aws](https://img.shields.io/website-up-down-green-red/http/cfn101.workshop.aws.svg)](https://cfn101.workshop.aws/)
+This repository provides all the resources referenced in the [CloudFormation](https://cfn101.workshop.aws/) workshop as
+well as the code used to build it.
 
-## Workshop site
+## Usage
+1. Clone the repository to your working directory or Download the ZIP file from GitHub.
+2. Open the downloaded files in your code editor or IDE of your choice.
 
-Go to [https://cfn101.workshop.aws](https://cfn101.workshop.aws/) to start a workshop.
+The working directory is located in CloudFormation templates and can be found in [code/workspace](code/workspace)
+where you can follow along and write your code to.
 
-## Developer Guide
+In the [code/solutions](code/solutions), you can find final template for each lab. This can be used as a reference,
+in case you get stuck and things don't work the way they should.
 
-This workshop is built with markdown as a static HTML site using [hugo](http://gohugo.io).
+## Local development
+To set up local development environment, please follow the instructions in
+[local development](docs/LOCAL_DEVELOPMENT.md) file.
```

Comment:
```suggestion
To set-up a local development environment for changing the workshop, please follow the instructions in
[local development](docs/LOCAL_DEVELOPMENT.md) file.
```

Clarify that this is for people wanting to change the workshop, not intended for people following it.